recipes/graphaligner/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: graphaligner
+  version: 1.0.0
+
+source:
+  url: https://github.com/maickrau/GraphAligner/files/2764381/GraphAligner.tar.gz
+  sha256: 6bbebd3c7373c2018245391c8ecdc97cc58e2afa82d3523e08afd0407df1bf63
+  patches: version.patch
+
+build:
+  number: 0
+  skip: False # [linux]
+  skip: True # [osx]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - jemalloc
+    - zlib
+    - mummer4
+    - boost
+    - libboost
+    - sparsehash
+    - pkg-config
+    - libdivsufsort
```

Comment:
Do you need to pin this? Is the library API stable?

Reply:
Turns out that libdivsufsort was getting statically linked even though the makefile suggests that it should be dynamically linked. I removed it from the run: section and it still seems to work.